.github/workflows/python-wheels.yml

```diff
@@ -48,10 +49,21 @@ jobs:
             maturin build --sdist --out wheelhouse
         if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.11'
 
-      - name: Build wheels
+      # By building 
+      - name: Build wheels from sdist
```

Comment:
Here it is not obvious why we need to manually unpack and do the build from sdist (when building from source would normally just work).
From outer context, I know that it is probably to avoid things like https://github.com/openwsn-berkeley/lakers/issues/289
Can you add a comment on that?

Reply:
Oh, there's an incomplete comment anyway; fixed.

Reply:
Redid the comment once because the original version was not spot-on.